README.md

```diff
@@ -67,33 +49,30 @@ seatoxin,seatoxin.fa,seatoxin-ref.fa,seatoxin_structures
 toxin,toxin.fa,toxin-ref.fa,toxin_structures
 ```
 
-Each row represents a set of sequences (in this case the seatoxin and toxin protein families) to be processed.
-
-`id` is the name of the set of sequences. It can correspond to the protein family name or to an internal id.
+Each row represents a set of sequences (in this case the seatoxin and toxin protein families) to be aligned.
```

Comment:
I think that explaining what each column represents (even if they are optional) as before would be clearer for users

Reply:
I was following the rnaseq structure and in the main they keep it super minimal and specify the columns in usage and this is what i tried to do - i could copy it here from usage but then it is maybe a bit redundant? I added some overview on the columns here bt not as secific as usage - let me know aht you think!